core/state_processor_test.go

```diff
@@ -383,8 +383,9 @@ func GenerateBadBlock(parent *types.Block, engine consensus.Engine, txs types.Tr
 		BaseFee:   baseFee,
 	}
 	if configExtra.IsApricotPhase4(header.Time) {
-		header.BlockGasCost = big.NewInt(0)
-		header.ExtDataGasUsed = big.NewInt(0)
+		ex := types.GetHeaderExtra(header)
```

Comment:
nit: not that I'm against this `ex` name, but not sure how I feel about it. I'd say let's go with `headerExtra` or `extra` for consistency